smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftDependency.kt

```diff
@@ -58,204 +58,40 @@ class SwiftDependency(
-            DistributionMethod.SPR,
-        )
-        val SMITHY_TIMESTAMPS = SwiftDependency(
-            "SmithyTimestamps",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../smithy-swift",
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
-        val SMITHY_XML = SwiftDependency(
-            "SmithyXML",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../smithy-swift",
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
-        val SMITHY_JSON = SwiftDependency(
-            "SmithyJSON",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../smithy-swift",
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
-        val SMITHY_FORM_URL = SwiftDependency(
-            "SmithyFormURL",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../smithy-swift",
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
-        val SMITHY_WAITERS_API = SwiftDependency(
-            "SmithyWaitersAPI",
-            "main",
-            "0.0.1",
-            "aws-sdk-swift",
-            "../../../smithy-swift",
-            "smithy-swift",
-            DistributionMethod.SPR,
-        )
+        val CLIENT_RUNTIME = smithySwiftDependency("ClientRuntime")
+        val SMITHY = smithySwiftDependency("Smithy")
+        val SMITHY_IDENTITY_API = smithySwiftDependency("SmithyIdentityAPI")
+        val SMITHY_IDENTITY = smithySwiftDependency("SmithyIdentity")
+        val SMITHY_RETRIES_API = smithySwiftDependency("SmithyRetriesAPI")
+        val SMITHY_RETRIES = smithySwiftDependency("SmithyRetries")
+        val SMITHY_HTTP_API = smithySwiftDependency("SmithyHTTPAPI")
+        val SMITHY_HTTP_AUTH_API = smithySwiftDependency("SmithyHTTPAuthAPI")
+        val SMITHY_HTTP_AUTH = smithySwiftDependency("SmithyHTTPAuth")
+        val SMITHY_CHECKSUMS_API = smithySwiftDependency("SmithyChecksumsAPI")
+        val SMITHY_CHECKSUMS = smithySwiftDependency("SmithyChecksums")
+        val SMITHY_STREAMS = smithySwiftDependency("SmithyStreams")
+        val SMITHY_EVENT_STREAMS_API = smithySwiftDependency("SmithyEventStreamsAPI")
+        val SMITHY_EVENT_STREAMS_AUTH_API = smithySwiftDependency("SmithyEventStreamsAuthAPI")
+        val SMITHY_EVENT_STREAMS = smithySwiftDependency("SmithyEventStreams")
+        val SMITHY_TEST_UTIL = smithySwiftDependency("SmithyTestUtil")
+        val SMITHY_READ_WRITE = smithySwiftDependency("SmithyReadWrite")
+        val SMITHY_TIMESTAMPS = smithySwiftDependency("SmithyTimestamps")
+        val SMITHY_XML = smithySwiftDependency("SmithyXML")
+        val SMITHY_JSON = smithySwiftDependency("SmithyJSON")
+        val SMITHY_FORM_URL = smithySwiftDependency("SmithyFormURL")
+        val SMITHY_WAITERS_API = smithySwiftDependency("SmithyWaitersAPI")
```

Comment:
All of the smithy-swift libraries have been DRYed up with the helper method below.